index.html

```diff
@@ -8,47 +8,355 @@
+            <img src="/images/Img4.png" alt="Mucyo" class="image" />
+            <div class="card">
+              <h1>Profesonal Art <br />Printing Data</h1>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="technology">
+                <ul>
+                  <li><a href="#"></a>html</li>
+                  <li class="boot2"><a href="#"></a>bootstrap</li>
+                  <li><a href="#"></a>Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="cardyellow"><h4>See project</h4></div>
+          </div>
+        </div>
+        <div id="content_one">
+          <div class="content_3">
+            <div>
+              <h1>About me</h1>
+              <p>
+                Hello I'am a Software Developer! I can help you build a product,
+                feature or website Look through some of my work and experience!
+                if you like what you see and have a project you need coded,
+                don't hestiate to contacta me.
+              </p>
+            </div>
+
+            <div class="btn_yellow" id="yellow">
+              <button type="submit">Get my Resume</button>
+            </div>
+
+            <div class="line5"></div>
+          </div>
+
+          <div class="bottom_link">
+            <div>
+              <h2>
+                <img src="images/Rectangle55.png" alt="" class="image55" />
+                Language
+              </h2>
+              <ul class="list">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+            </div>
+            <div>
+              <h2>
+                <img src="images/Rectangle56.png" alt="" class="image55" />
+                Frameworks
+              </h2>
+              <ul class="list">
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
+            <div>
+              <h2>
+                <img src="images/Rectangle57.png" alt="" class="image55" />
+                Skills
+              </h2>
+              <ul class="list">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="footer_form">
+        <div class="content_4">
+          <p>
+            I'm always interested in <br />
+            hearing about new projects, <br />so if you'd like to<br />
+            chat please get in.
+          </p>
+        </div>
+        <div class="form_view">
```

Comment:
> This section should cover the entire space and equal the section above.
![Screenshot from 2021-08-20 12-43-56](https://user-images.githubusercontent.com/65501535/130214609-8ed21d6a-cd3b-4b82-9de8-71c6473b76f4.png)

Please consider stretching out this section to fit and match the other sections. Focus mainly on its padding/margin